drivers/sensor/lsm6dsl/lsm6dsl.c

```diff
@@ -774,6 +774,22 @@ static int lsm6dsl_init_chip(const struct device *dev)
 		return -EIO;
 	}
 
+	if (data->hw_tf->update_reg(dev,
+				    LSM6DSL_REG_CTRL6_C,
+				    LSM6DSL_MASK_CTRL6_C_XL_HM_MODE,
+				    (1 << LSM6DSL_SHIFT_CTRL6_C_XL_HM_MODE)) < 0) {
+		LOG_DBG("failed to disable accelerometer high performance mode");
```

Comment:
The debug log info is the same for both cases which is difficult to distinguish which one failed. Could you fix that?

Reply:
There are two diffrent debug log:
LOG_DBG("failed to disable accelerometer high performance mode");
and

Reply:
Ops, right. I didn't notice that. It looks good to me.